pkg/kubeapiserver/authenticator/config.go

```diff
@@ -212,7 +207,27 @@ func (config Config) New() (authenticator.Request, *spec.SecurityDefinitions, er
 		authenticator = union.NewFailOnError(authenticator, anonymous.NewAuthenticator())
 	}
 
-	return authenticator, &securityDefinitions, nil
+	return authenticator, &securityDefinitionsV2, securitySchemesV3, nil
+}
+
+func populateOpenAPISecurity(v2 spec.SecurityDefinitions, v3 spec3.SecuritySchemes) {
```

Comment:
probably easier to understand to keep this inlined if this function is only going to be used in a single place

Reply:
After moving everything inline the function didn't seem that useful and I've just removed the function and placed everything inline.